app/models/attachment.rb

```diff
@@ -34,7 +34,7 @@ class Attachment < ApplicationRecord
   belongs_to :updated_by_user, class_name: 'User'
 
   has_attached_file :attachment,
-            path: ':rails_root/public/system/attachments/:hash.:extension',
+            path: ":rails_root/public/system/#{Rails.env}/attachment/:hash.:extension",
```

Comment:
That path value still gets used by S3, changing the target path breaks all existing image links.

Reply:
really? S3 uses this? Surely it's just for dev and test env, no? The rails_root/public/system path is in the Rails app, not the S3 server isn't it? Or else I'm misunderstanding something, (not the first time)